diff --git a/src/Codeception/Test/Cest.php b/src/Codeception/Test/Cest.php
index c644ed2..5426bf3 100644
--- a/src/Codeception/Test/Cest.php
+++ b/src/Codeception/Test/Cest.php
@@ -150,7 +150,7 @@ protected function executeTestMethod($I)
         $this->invoke($this->testMethod, [$I, $this->scenario]);
     }
 
-    public function toString()
+    public function toString(): string
     {
         return sprintf('%s: %s', ReflectionHelper::getClassShortName($this->getTestClass()), Message::ucfirst($this->getFeature()));
     }
diff --git a/src/Codeception/Test/Test.php b/src/Codeception/Test/Test.php
index f5e8426..99c9f0a 100644
--- a/src/Codeception/Test/Test.php
+++ b/src/Codeception/Test/Test.php
@@ -55,7 +55,7 @@ abstract class Test implements TestInterface, Interfaces\Descriptive
      *
      * @return mixed
      */
-    abstract public function toString();
+    abstract public function toString(): string;
 
     /**
      * Runs a test and collects its result in a TestResult instance.
